torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -723,6 +723,9 @@ def rewrite_node(node):
                 # and this can then go to ReplaceMoreTt class.
                 output_size = node.meta["val"].size()
                 rank = len(output_size)
+                # TODO(tt-metal#15165): ttnn.permute > 4D shape is not supported yet
```

Comment:
We will need to resolve https://github.com/tenstorrent/tt-metal/issues/15165 to remove this fallback